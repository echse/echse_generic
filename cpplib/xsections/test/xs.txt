# bla
x y  elev rough
0 0.  3.  30
0 0.  0.  30
0 10. 0.  30
0 10. 3.  30

