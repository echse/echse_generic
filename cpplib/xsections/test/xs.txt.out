# River:           river_1
# Reach:           reach_1
# Station:         1
# 3D-data?:        1
# Number of zones: 1
xcoord	ycoord	offset	elevation	roughness	zone_index
0	0	0	3	30	0
0	0	0	0	30	0
0	10	10	0	30	0
0	10	10	3	30	0
